entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/ash
+
+set -euo pipefail
+
+chown -R pypiserver:pypiserver /data
+# Set the setgid bit so anything added here gets associated with the
+# pypiserver group
+# chmod g+s /data/packages
+
+if [ "$@" = "" ]; then
+    # default CMD
+    echo "Set default option '/data/packages'"
+    set -- " /data/packages"
+else
+    # 
+    echo "Using custom CMD: $@"
+fi
```

Comment:
I'm a little concerned that this update could break people's existing commands. If someone were relying on `packages` being provided by default, and were adding extra commands on top of that, it seems like this might break, unless I'm missing something?

e.g. if someone were running `docker run pypiserver -- log-file log.out`?

Reply:
I tested a little bit. So, as far as I can tell, the above command `docker run pypiserver --log-file log.out` won't work in the current implementation, because the `packages` directory is not added by default as soon as docker gets additional parameters.
```
Error: while trying to list root(/home/pypiserver/packages): [Errno 2] No such file or directory: '/home/pypiserver/packages'
```
Ok, there is a default `packages` path: `/home/pypiserver/packages`, but it does not exist in the docker container.

---

The default `packages` path is added because of
```
CMD ["packages"]
```
However, this only works in case no additional docker commands are given when the container is started.
The only option, that's added every time is the port, because - in the original implementation - it is part of the `ENTRYPOINT`:
```
ENTRYPOINT ["pypi-server", "-p", "8080"]
```

---

The given command in the Dockerfile's `CMD` will be overwritten when doing
```
docker run pypiserver --log-file log.out
```

The command should be
```
docker run pypiserver --log-file log.out packages
```
to make it work properly.

---

I think the new implementation (contained in this PR) behaves in exactly the same way the original implementation does.

Of course it is possible to always add the default path, but I think, this will get confusing and can, in the end, actually affect user's commands.
What do you think?

Reply:
Thanks for digging in on that! I think you're right. Any specification of additional arguments on the current dockerfile would override the default `packages` argument, so I think this will behave exactly like it did before.